packages/@lwc/template-compiler/src/codegen/codegen.ts

```diff
@@ -628,16 +628,22 @@ export default class CodeGen {
                 for (const attribute of node.attributes) {
                     const { name, value } = attribute;
                     if (isExpression(value)) {
-                        this.staticExpressionMap.set(
-                            attribute,
-                            `${STATIC_PART_TOKEN_ID.ATTRIBUTE}${partId}:${name}`
-                        );
-                        attributeExpressions.push(
-                            t.property(
-                                t.literal(name),
-                                bindAttributeExpression(attribute, node, this, false)
-                            )
-                        );
+                        let partToken = '';
+                        if (name === 'style') {
+                            partToken = `${STATIC_PART_TOKEN_ID.STYLE}${partId}:`;
```

Comment:
The dangling `:` feels redundant here. We could save a bit of space by removing it.

Reply:
[Removed](https://github.com/salesforce/lwc/pull/4084/commits/52042750726109d0dbe44f0f641a86c4ce486167#diff-fa7a672e9062c62e45e9a00887fcf61da036aa12fe10949398d018204668fb15R633) and updated the [SSR parsing](https://github.com/salesforce/lwc/pull/4084/commits/52042750726109d0dbe44f0f641a86c4ce486167#diff-9d36a520650dfa1dd13874ab3870b6f688303b9e5860066178098aadcf45a860R136-R147).